druid/src/tests/mod.rs

```diff
@@ -332,7 +334,7 @@ fn simple_lifecyle() {
         harness.send_initial_events();
         assert_matches!(record.next(), Record::L(LifeCycle::WidgetAdded));
         assert_matches!(record.next(), Record::E(Event::WindowConnected));
-        assert_matches!(record.next(), Record::E(Event::WindowSize(_)));
+        assert_matches!(record.next(), Record::None);
```

Comment:
Not sure if this is important to record `WindowSize` here. It can't be recorder due to `WindowDesc::new` now implicitly wrap root widget with `FocusScope`